scripts/seurat_alignment_qc.Rmd

```diff
@@ -58,8 +55,16 @@ suppressPackageStartupMessages({
   library(stringr)
   library(ggpmisc)
   library(ggrepel)
-  theme_set(theme_bw())
-  
+  library(miQC)
+  library(scater)
+  library(Seurat)
+  library(SingleCellExperiment)
+  library(flexmix) # to estimate mixtureModel for miQC
+  library(irlba) # this solves the issue with RunUMAP code chunk
+  library(SeuratWrappers)
```

Comment:
So far, in my testing, I don't think you need this but...

Reply:
Then let's drop it :)